src/nvim/eval/funcs.c

```diff
@@ -3507,6 +3507,29 @@ static void f_getloclist(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   get_qf_loc_list(false, wp, &argvars[1], rettv);
 }
 
+
+/// "getmarklist()" function
+static void f_getmarklist(typval_T *argvars, typval_T *rettv, FunPtr fptr)
+{
+  buf_T *buf = NULL;
```

Comment:
buf is set to an unused value. Declare and initialize the variable on the same line below.

Reply:
should be fixed in 6764086